script/GenKey.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# usage: ./GenKey.sh [<directory_name>]
+
+# get blpapi-http root directory
+DNAME=${1:-temp}
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && mkdir "$DNAME" && cd "$DNAME" && pwd )
```

Comment:
This can be simplified a bit and there is no need to use `cd`.  http://mywiki.wooledge.org/BashFAQ/035

I would recommend having the usage be like this:

./gen_keys.sh <dir_name>, where the directory name is the current working directory if the directory is not specified.